TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift

```diff
@@ -109,7 +137,12 @@ extension PushFactory: PushFactoryProtocol {
     }
   }
   
-  func verifyFactor(withSid sid: String, success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func verifyFactor(
+    withSid sid: String,
+    allowIphoneMigration: Bool,
```

Comment:
We should read the value for allowIphoneMigration from the factor instead of passing this as parameter